practice/practice-02/src/sass/abstract/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$primary-font: "Sofia Pro";
+$black: #000;
+$white: #fff;
+$primary-color: #f54748;
+$secondary-color: #fdc55e;
+$fs-md: 18px;
```

Comment:
For all default set, you should list all and define it first of all. Don't define it when you use it

Reply:
Added.